admin/src/Main.elm

```diff
@@ -92,13 +87,16 @@ init flags url key =
     let
         route =
             Routes.toRoute url
+
+        fetchToday =
+            Naive.now
     in
-    initWith key route
+    ( Starting key route, Task.perform FetchedToday fetchToday )
```

Comment:
I have no clue of elm syntax :') but why can't this be just `(Starting key Routes.toRoute url, Task.perform FetchedToday Naive.now)`

Reply:
It could be `Starting key (Routes.toRoute url)`. I just find it cleaner to read this way.